### PR TITLE
feat: add mindspore to cookbook

### PR DESCRIPTION
**Pull request by bwanglzu**
Add mindspore to Jina 2.0 cookbook. tested .

### CODE REVIEW

File: .github/2.0/cookbooks/Executor.md [hunk 1]
```diff
@@ -514,5 +514,33 @@ class PaddleMwuExecutor(Executor):
...
+import mindspore.ops as ops
+import mindspore.context as context
+
+from jina import Executor, requests
+
+
+class MindsporeMwuExecutor(Executor):
+    def __init__(self, **kwargs):
+        super().__init__()
+        context.set_context(mode=context.PYNATIVE_MODE, device_target="CPU")
```
Comment (nan-wang) [inline .github/2.0/cookbooks/Executor.md:531]:
```suggestion
        context.set_context(mode=context.PYNATIVE_MODE, device_target='CPU')
```

**Review by nan-wang — CHANGES_REQUESTED** [review]
minor change

File: .github/2.0/cookbooks/Executor.md [hunk 2]
```diff
@@ -629,3 +629,38 @@ class TfMobileNetEncoder(Executor):
...
+    def __init__(self, **kwargs):
+        super().__init__()
+        context.set_context(mode=context.PYNATIVE_MODE, device_target='CPU')
+        self.dims = 5
+        self.encoding_mat = Tensor(np.random.rand(self.dims, self.dims))
+
+    @requests
+    def encode(self, docs, **kwargs):
+        for doc in docs:
+            input_tensor = Tensor(
```
Comment (JoanFM) [inline .github/2.0/cookbooks/Executor.md:658]: can u make it not pass black? I think it looks leaner?

File: .github/2.0/cookbooks/Executor.md [hunk 3]
```diff
@@ -629,3 +629,38 @@ class TfMobileNetEncoder(Executor):
...
+import mindspore.context as context
+
+from jina import Executor, requests
+
+
+class MindsporeMwuExecutor(Executor):
+    def __init__(self, **kwargs):
+        super().__init__()
+        context.set_context(mode=context.PYNATIVE_MODE, device_target='CPU')
+        self.dims = 5
```
Comment (JoanFM) [inline .github/2.0/cookbooks/Executor.md:652]: you do not need self.dims

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Review by JoanFM — APPROVED** [review]

File: .github/2.0/cookbooks/Executor.md [hunk 4]
```diff
@@ -629,3 +629,33 @@ class TfMobileNetEncoder(Executor):
...
+
+# MindSpore
+
+The code snippet below takes ``docs`` as input and perform matrix multiplication with ``self.encoding_matrix``.
+It leverages Mindspore ``Tensor`` conversion and operation.
+Finally, the ``embedding`` of each document will be set as the multiplied result as ``np.ndarray``.
+
+```python
+import numpy as np
+from mindspore import Tensor  # mindspre 1.2.0
```
Comment (JoanFM) [inline .github/2.0/cookbooks/Executor.md:641]:
```suggestion
from mindspore import Tensor  # mindspore 1.2.0
```

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Review by JoanFM — APPROVED** [review]

**Pull request merged**